config/default/kustomization.yaml

```diff
@@ -20,16 +20,18 @@ commonLabels:
   app.kubernetes.io/managed-by: kustomize
   app.kubernetes.io/part-of: manual-deployment
 
+
 patches:
-- patch: |-
-    - op: replace
-      path: /metadata/name
-      value: lifecycle-manager-watcher-gateway
-  target:
-    group: networking.istio.io
-    version: v1beta1
-    kind: Gateway
-    name: watcher-gateway
+# [WATCHER]
```

Comment:
for the default profile, we don't need watcher, right? It's better we remove it completely.

Reply:
removed it